feature-libs/organization/administration/components/unit/details/unit-details.component.ts

```diff
@@ -30,5 +31,10 @@ export class UnitDetailsComponent {
   );
   isInEditMode$ = this.itemService.isInEditMode$;
```

Comment:
we can use `readonly` to prevent modification after assignment

Reply:
isInEditMode$ was not added by me as part of this PR.. it is an already available variable. Hence not changing it.